pkg/plugins/optional/grafana/v1alpha/scaffolds/edit.go

```diff
@@ -41,15 +47,122 @@ func (s *editScaffolder) InjectFS(fs machinery.Filesystem) {
 	s.fs = fs
 }
 
+func fileExist(configFilePath string) bool {
+	if _, err := os.Stat(configFilePath); os.IsNotExist(err) {
+		return false
+	}
+	return true
+}
+
+func loadConfig() ([]templates.CustomMetricItem, error) {
+	// TODO: Is it good to hard code the config file path here?
+	configPath := "grafana/custom-metrics/config.yaml"
```

Comment:
For now, let's make this constant. We can look into allowing users to specify this later. Instead of declaring it locally, let's make this a global variable:
```
const configFilePath := "grafana/custom-metrics/config.yaml"
```